resultsdbupdater/utils.py

```diff
@@ -282,7 +282,7 @@ def handle_ci_umb(msg):
     if item_type == 'productmd-compose':
         architecture = system['architecture']
         variant = system.get('variant')
-        compose_id = msg_body['artifact']['compose_id']
+        compose_id = msg['headers']['compose_id']
```

Comment:
hmm hmm :) the new field will be called 'id' and it would be fine to still use artifact instead of headers. We wanted to deprecate headers.

Reply:
note that after speaking with RTT, what about to accept for now both, something like:
```
compose_id = get(msg_body['artifact'], 'id', None) or msg_body['artifact']['compose_id']
```